### ISSUE TITLE
End marker never shows

### ISSUE THREAD

**Issue by nielsrune** [OFFLINE-GmbH/multi-timeline.js#2]
Both the source and the minified file causes the end marker to never show.
End marker shows correctly if using the (older?) version from the example page.

Seems to stem from the following line which is commented out.

https://github.com/OFFLINE-GmbH/multi-timeline.js/blob/6d4cf058f31f1947a0ec1d359402e9c2e50618dd/src/js/multi-timeline.js#L348

Also, shouldn't this be "**b**order" and not "order"?
https://github.com/OFFLINE-GmbH/multi-timeline.js/blob/6d4cf058f31f1947a0ec1d359402e9c2e50618dd/src/scss/multi-timeline.scss#L111

**Comment by tobias-kuendig** [comment]
Good catch, that should definitively be `border`. 

I don't think the commented out line is the cause of the problem. If you specify an `end` value and don't set the `markerDateFormat` to `false` you should get the end marker. If not, this is the place i'd start to debug:
https://github.com/OFFLINE-GmbH/multi-timeline.js/blob/master/src/js/multi-timeline.js#L464

If you find the cause of the problem please feel free to submit a PR with a bugfix.

**Comment by nielsrune** [comment]
Thank you for your quick reply.

In my test page I did specify an `end` value and set `markerDateFormat` to a valid date format. No end marker is shown even with a fairly wide time span.

Retrieving the js-file from the [demo page] and switching to this file in my test page's head causes the end marker to show correctly.

I've already did some debugging and started from line 464 as you also suggest.
My result this far is that the end marker div is created but gets removed again here:
https://github.com/OFFLINE-GmbH/multi-timeline.js/blob/6d4cf058f31f1947a0ec1d359402e9c2e50618dd/src/js/multi-timeline.js#L352

This is inside the condition above:
https://github.com/OFFLINE-GmbH/multi-timeline.js/blob/6d4cf058f31f1947a0ec1d359402e9c2e50618dd/src/js/multi-timeline.js#L351

Setting a breakpoint and testing the `outerWidth` shows me, that this value is never higher than the 140 threshold. Even a time span nearly as wide as a timeline defined for a whole month resolves to a `outerWidth` value of approx. 38.

Switching to the 'working' js from the [demo page] gives a `outerWidth` value of more than 1900, thus show (ie. not removing) the end marker div.

I might test a few things more, but for now I haven't figured out why the `outerWidth` is too low.

This is the status so far. I will be happy to do a pull request if I find the answer.

[demo page]: https://offline-gmbh.github.io/multi-timeline.js/
